--- a/Celeste.Mod.mm/Mod/Everest/Everest.Flags.cs
+++ b/Celeste.Mod.mm/Mod/Everest/Everest.Flags.cs
@@ -83,7 +83,7 @@ namespace Celeste.Mod {
                 PreferThreadedGL = IsXNA;
 
                 SupportRuntimeMods = true;
-                SupportUpdatingEverest = true;
+                SupportUpdatingEverest = false;
             }
 
         }
--- a/Celeste.Mod.mm/Mod/Core/CoreModuleSettings.cs
+++ b/Celeste.Mod.mm/Mod/Core/CoreModuleSettings.cs
@@ -217,7 +217,7 @@ namespace Celeste.Mod.Core {
 
         [SettingInGame(false)]
         [SettingIgnore] // TODO: Show as advanced setting.
-        public bool? WhitelistFullOverride { get; set; } = null;
+        public bool? WhitelistFullOverride { get; set; } = true;
 
         [SettingInGame(false)]
         [SettingIgnore] // TODO: Show as advanced setting.
